packages/transformer/src/BigMap.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { Id64String } from "@itwin/core-bentley";
```

Comment:
you have a docs error because you don't have a docs tag for this module:
```
Topic BigMap does not have a @docs-group-description in imodel-transformer.ts. Please add a description.
```
Can we fix that?